specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2019-01-01-preview/examples/alertRuleTemplates/GetAlertRuleTemplates.json

```diff
@@ -33,9 +33,7 @@
               "requiredDataConnectors": [
                 {
                   "connectorId": "AWS",
-                  "dataTypes": {
-                    "AWSCloudTrail": "Exist"
-                  }
+                  "dataTypes": ["AWSCloudTrail"]
```

Comment:
the format should be
```
"dataTypes": [
     "AWSCloudTrail"
]
```

Reply:
@yoshkedi please fix it

Reply:
ok, I did. 
still prettier fails. 
how can I see the lines that failed it?

Reply:
ok fixed it, can you please merge?